the-collab-lab/tcl-63-smart-shopping-list
# Change Button Color Contrast to Meet Accessibility Requirement

## Description


All buttons currently have a light blue background which doesn't pass Lighthouse's accessibility requirement.

## Related Issue
Accessibility


## Acceptance Criteria



- [x] Modify button background color to a darker black to meet Lighthouse's accessibility requirement.

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|    | :sparkles: New feature     |
| ✓    | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

## Updates

### Before


![lighthouse-accessibility-test-before](https://github.com/the-collab-lab/tcl-63-smart-shopping-list/assets/24996005/201351ed-fc62-46bc-91fe-307fce3da6c8)

### After


![lighthouse-accessibility-test-after](https://github.com/the-collab-lab/tcl-63-smart-shopping-list/assets/24996005/488bb7fc-e853-47ea-91c4-b3de1c1ba3ce)

## Testing Steps / QA Criteria


To pull our branch locally, run `git pull origin btn-color-contrast`
Next, run `git checkout btn-color-contrast`
Launch the app with `npm start`
Install `Lighthouse` Chrome extension if you don't have it already
Go to the `Add Item` page, or any page with the button. Open the console on the browser, go to `Lighthouse` tab and Generate a Lighthouse report on the "Accessibility" category, expect to see a rating of 100.
